### DISCUSSION TITLE
Thanks for making this

### DISCUSSION THREAD

**Discussion by Posandu** [Jelmerro/Vieb#342] [General]
The best browser I used. the website UI needs to be improved through

**Comment by Jelmerro** [comment]
Thanks for the compliment, really nice to hear so many people enjoying Vieb. What about the website needs to be improved, I can improve it and/or make a repository for the website so others can do so. Does this also apply to the application itself? Because Vieb supports custom CSS using colorschemes :wink:

**Comment by Posandu** [comment]
> make a repository for the website so others can do so

Yes, so we can contribute

**Comment by Jelmerro** [comment]
Here you go @Posandu: https://github.com/Jelmerro/vieb.dev
Feel free to contribute any improvements you can think of, as long as we keep it lightweight.
